model/model.go

```diff
@@ -26,7 +27,10 @@ import (
 // Also used by the Makefile for releases.
 // Default to the version of the program
 // but may be overridden for demo/documentation mode.
-var Version = "0.9.0"
+var (
```

Comment:
Can you put this new variable on its own line?
Otherwise, the Makefile will fail for target 'tag'
https://github.com/clearlinux/clr-installer/blob/master/Makefile#L201

Reply:
Sure.